pimd/pim_nb_config.c

```diff
@@ -2120,6 +2120,10 @@ int lib_interface_pim_address_family_multicast_boundary_oil_modify(
 	case NB_EV_PREPARE:
 		break;
 	case NB_EV_APPLY:
+#if PIM_IPV == 6
+		/* TBD Depends on MLD data structure changes */
+		break;
+#endif
```

Comment:
the remaining code will still execute for both V4 and V6, since it is not under IPV4

Reply:
I have break the condition, so it should come out of switch